packages/block-library/src/post-content/edit.js

```diff
@@ -32,9 +32,10 @@ function ReadOnlyContent( { userCanEdit, postType, postId } ) {
 			<Warning>{ __( 'This content is password protected.' ) }</Warning>
 		</div>
 	) : (
-		<div { ...blockProps }>
-			<RawHTML key="html">{ content?.rendered }</RawHTML>
-		</div>
+		createElement( 'div', {
```

Comment:
You don't need to use `createElement`. Just append the `dangerouslySetInnerHTML` to the existing `div`.

Reply:
Of course 🤦 

Done!